.dockerignore

```diff
@@ -0,0 +1,10 @@
+# Ignore everything
+*
+
+# Whitelist method
```

Comment:
ok, but now you just inverted the original issue. 
If you add a CSS file, what now? You still need to update this file to include it in the whitelist, right? 

Find the root cause and fix that.